specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/keys.json

```diff
@@ -0,0 +1,557 @@
+            "P-256K"
+          ],
+          "x-ms-enum": {
+            "name": "JsonWebKeyCurveName",
+            "modelAsString": true
+          }
+        },
+        "keyUri": {
+          "type": "string",
+          "description": "The URI to retrieve the current version of the key.",
+          "readOnly": true
+        },
+        "keyUriWithVersion": {
+          "type": "string",
+          "description": "The URI to retrieve the specific version of the key.",
+          "readOnly": true
+        }
+      },
+      "description": "The properties of the key."
+    },
+    "KeyAttributes": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Determines whether or not the object is enabled."
+        },
+        "nbf": {
+          "x-ms-client-name": "NotBefore",
+          "type": "integer",
+          "format": "int64",
+          "description": "Not before date in seconds since 1970-01-01T00:00:00Z."
+        },
+        "exp": {
+          "x-ms-client-name": "Expires",
+          "type": "integer",
+          "format": "int64",
+          "description": "Expiry date in seconds since 1970-01-01T00:00:00Z."
+        },
+        "created": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Creation time in seconds since 1970-01-01T00:00:00Z."
+        },
+        "updated": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Last updated time in seconds since 1970-01-01T00:00:00Z."
+        },
+        "recoveryLevel": {
+          "type": "string",
+          "description": "The deletion recovery level currently in effect for the object. If it contains 'Purgeable', then the object can be permanently deleted by a privileged user; otherwise, only the system can purge the object at the end of the retention interval.",
+          "enum": [
+            "Purgeable",
+            "Recoverable+Purgeable",
+            "Recoverable",
+            "Recoverable+ProtectedSubscription"
+          ],
+          "x-ms-enum": {
+            "name": "DeletionRecoveryLevel",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "x-nullable": false
+        }
+      },
+      "description": "The attributes of the key."
+    },
+    "KeyCreateParameters": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags that will be assigned to the key."
+        },
+        "properties": {
+          "$ref": "#/definitions/KeyProperties",
+          "description": "The properties of the key to be created."
+        }
+      },
+      "description": "The parameters used to create a key.",
+      "required": [
+        "properties"
+      ],
+      "x-ms-azure-resource": true
+    },
+    "Key": {
```

Comment:
Even flattening, this doesn't match a `KeyBundle`: https://github.com/Azure/azure-rest-api-specs/blob/1eeb7b57a444daae73dcadf1961f272693778b65/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.2/keys.json#L1197-L1221

Shouldn't it?